README.md

```diff
@@ -28,8 +28,13 @@ $ net use a: \\192.168.99.100\avalon
 
 ### Development
 
-Use the overrides to mount your local development directory in place of the deployed version.
+For development [Git](https://git-scm.com/) is required.
 
 ```bash
-$ docker-compose -f docker-compose.yml -f docker-compose-dev.yml up
+$ git clone https://github.com/getavalon/docker.git --recursive
+$ cd docker
+$ docker build . -t avalon/latest
```

Comment:
Think you mean `getavalon/docker:latest`, where `getavalon` is our Docker Hub org and `latest` is the version. Though for development, you can omit `latest` altogether, it's mostly relevant for users.